.githooks/pre-commit

```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env bash
 
-cargo make check-rs-format && \
-cargo make check-ts-format
+# Run check-rs-format if Rust source files were modified.
+git status | egrep "*.rs"
+if [ $? -eq 0 ]; then
+    cargo make check-rs-format
+    ret=$?
+    if [ $ret -ne 0 ]; then
+        exit $ret
+    fi
+fi
+
+# Run check-rs-format if Typescript source files were modified.
+git status | egrep "*.ts"
```

Comment:
📣 `check-ts-format` actually checks few other formats. Maybe we should rename it to `check-prettier`.
```suggestion
git status | egrep "*.{ts,json,yml}"
```

Reply:
Hmm, now we have one more duplication of the file extensions we want to check for formatting / format automatically.

Any ideas on how those could be unified to a single place? Currently, the other two usages are in `package.json`.